.github/workflows/release_next.yml

```diff
@@ -43,7 +43,7 @@ jobs:
       - name: release status
         id: get_release_status
         run: |
-          status=$(sentry-cli releases list | grep 'cdktf-cli-${{ steps.get_version.outputs.version }} ' && echo 'released' || echo 'unreleased')
+          status=$(sentry-cli releases info 'cdktf-cli-${{ steps.get_version.outputs.version }}' && echo 'released' || echo 'unreleased')
```

Comment:
Did it work? What is this fixing exactly?

When I last looked at the `info` command, it didn't contain the `unreleased` state of releases already 😅 (But we don't really need it here, but rather in the other check later in the release workflow)
<img width="818" alt="image" src="https://user-images.githubusercontent.com/1112056/205901446-c3faf2af-149d-41d6-b06e-ba04428f4567.png">
<img width="898" alt="image" src="https://user-images.githubusercontent.com/1112056/205901979-3c70f67a-ef16-44e8-83ed-800d90b73491.png">


Also, this will still result in a multiline string that will be echoed into `$GITHUB_OUTPUT`, so I don't think it actually fixes the error we're seeing for releases that already exist 😬

Reply:
locally it did work, but as you said it does not tell us anything about the released status. I'm trying to fix the mulit-line issue